stdlib/public/core/Integers.swift.gyb

```diff
@@ -896,15 +900,15 @@ def unsafeOperationComment(operator):
   /// - Returns: The sum of this value and `rhs`.
 """,
         '-': """\
-  /// Returns the difference of this value and the given value without checking
-  /// for arithmetic overflow.
+  /// Returns the difference obtained by subtracting the given value from this
```

Comment:
Not that there would have been terrible confusion, but we could stand to clarify which value is the subtrahend, so I did.